libs/core/langchain_core/tracers/langchain.py

```diff
@@ -191,75 +183,69 @@ def _update_run_single(self, run: Run) -> None:
             log_error_once("patch", e)
             raise
 
-    def _submit(self, function: Callable[[Run], None], run: Run) -> None:
-        """Submit a function to the executor."""
-        if self.executor is None:
-            function(run)
-        else:
-            self._futures.add(self.executor.submit(function, run))
-
     def _on_llm_start(self, run: Run) -> None:
         """Persist an LLM run."""
         if run.parent_run_id is None:
             run.reference_example_id = self.example_id
-        self._submit(self._persist_run_single, _copy(run))
+        self._persist_run_single(_copy(run))
```

Comment:
We candelete the `_copy()` function now since we are converting to a dict anyway